web_src/js/modules/worker.js

```diff
@@ -0,0 +1,8 @@
+const {appSubUrl} = window.config;
+
+export function logoutFromWorker() {
+  // wait for a while because other requests (eg: logout) may be in the flight
+  setTimeout(() => {
+    window.location.href = `${appSubUrl}/`;
+  }, 5000);
+}
```

Comment:
```suggestion
import {sleep} from '../utils.js';

export async function logoutFromWorker() {
  // wait for a while because other requests (eg: logout) may be in the flight
  await sleep(5000);
  window.location.href = `${appSubUrl}/`;
}
```

Reply:
Thought it does raise the problem with "no await", so I'm fine either way.

Reply:
If there is no serious problem, I won't keep suggesting contributors to follow my way.